prometheus_api_client/metric_range_df.py

```diff
@@ -24,6 +24,9 @@ class MetricRangeDataFrame(DataFrame):
                  default to list of labels + "timestamp" + "value" if not provided.
     :param dtype: (dtype) default None. Data type to force. Only a single dtype is allowed. If None, infer.
     :param copy: (bool) default False. Copy data from inputs. Only affects DataFrame / 2d ndarray input.
+    :param ts_floats_to_datetime: (bool) default True. Convert the timestamps returned by prometheus
```

Comment:
I'd suggest either:
- `ts_as_datetime` because it's shorter and you don't need to think about what prom returns
- `ts_float_to_datetime` to not mix plural (floatS) with singular (datetime).

Reply:
Updated, thanks for the review! Lmk if you have more suggestions :)